packages/components/src/components/as-dropdown/as-dropdown.tsx

```diff
@@ -56,6 +65,16 @@ export class Dropdown {
   @State() private isOpen: boolean = false;
   @State() private selectedOptionObject: DropdownOption = {};
 
+  /**
+   * Closes the list, useful in case you need to customize {onClickOutside}
+   *
+   * @memberof Dropdown
+   */
+  @Method()
+  public async closeList() {
```

Comment:
I think this is not needed

Reply:
If you override onClickOutside, the dropdown will not close anymore, because we're removing the listener when it changes. 

This method is a way for the user to close the list whenever they think is necessary, for example, after doing something on their custom onClickOutside.

Reply:
This new method is public but it's not documented in the reference.